docs/how_to/create_a_quantum_circuit.rst

```diff
@@ -0,0 +1,167 @@
+
+########################
+Create a quantum circuit
+########################
+
+This guide shows how to initialize a quantum circuit.
+
+There are two ways to create a :class:`~.QuantumCircuit` object:
+
+* Specifying the number of qubits and bits.
+* Creating :class:`~.QuantumRegister`\ s and :class:`~.ClassicalRegister`\ s.
+
+Create from number of qubits and bits
+=====================================
+
+In order to create a :class:`~.QuantumCircuit` by only specifying the number of bits and qubits, you need to follow these steps.
+
+.. testcode::
+
+    from qiskit import QuantumCircuit
+
+    # Initialize number of qubits and classical bits
+    n_qubits = 3
+    n_bits = 2
+
+    # Create the circuit
+    qc = QuantumCircuit(n_qubits, n_bits)
+    print(qc.draw())
```

Comment:
```suggestion
    qc.draw(output='text')
```

Reply:
I had to include the `print` because else `doctest` would interpret that as "no output"